### PR TITLE
[RW-636] Add live user validation for password

### PR DESCRIPTION
**Pull request by blrubenstein**
Version is deployed here: https://rw-636-dot-all-of-us-workbench-test.appspot.com

### CODE REVIEW

File: ui/src/app/views/account-creation/component.ts [hunk 1]
```diff
@@ -101,6 +90,34 @@ export class AccountCreationComponent {
...
+    // fields case.
+    if (this.password === undefined || this.passwordAgain === undefined
+      || this.password.length === 0 || this.passwordAgain.length === 0) {
+        return false;
+    }
+    return !(this.password === this.passwordAgain);
+  }
+
+  get showPasswordLengthError() {
+    // We do not want to show errors if nothing is typed yet. This is caught by the required
```
Comment (calbach) [inline ui/src/app/views/account-creation/component.ts]: I don't think I like the UX of live password validation with negative feedback. The annoyance here is that even if I have a valid password in mind, the UI unjustly complains at me halfway through typing it.

If we're going to do live validation, I think the better UX would be to use positive feedback on satisfying requirements, e.g. pw requirements are listed in bullet form, and you get a checkmark once you satisfy each.

Happy to defer to UX on this, it's just kind of a pet peeve of mine in web UIs.

Comment (blrubenstein) [reply ui/src/app/views/account-creation/component.ts]: Tagging @fknabe in since he filed the ticket.

Comment (blrubenstein) [reply ui/src/app/views/account-creation/component.ts]: That said, for feedback purposes, I think I agree that positive feedback would be better.

Comment (rushtong) [reply ui/src/app/views/account-creation/component.ts]: I'm really glad CH brought this up - it is a general concept we should keep in mind for all of our UIs.

Comment (blrubenstein) [reply ui/src/app/views/account-creation/component.ts]: Also going to tag @lstrano-broad for his thoughts.

Comment (lstrano-broad) [reply ui/src/app/views/account-creation/component.ts]: Agree CH, so let's acknowledge any positive/error feedback once the user goes to the next field - that would assume the user is done typing.

Comment (blrubenstein) [reply ui/src/app/views/account-creation/component.ts]: @lstrano-broad lets meet in person to go through this?

Comment (lstrano-broad) [reply ui/src/app/views/account-creation/component.ts]: Sure

Comment (blrubenstein) [reply ui/src/app/views/account-creation/component.ts]: Have met with Lou, closing this out for favor of more work to be done. Should have a new PR up soonish.

**Pull request closed without merging**